server/api/event/models.py

```diff
@@ -14,8 +14,11 @@ class Event(SoftDeleteModel):
     start_time = models.DateTimeField()
     end_time = models.DateTimeField(null=True, blank=True)
     location = models.CharField(max_length=200)
+
+    payment_link = models.CharField(max_length=200, blank=True)
```

Comment:
Can you also add null=true here, so it is an optional field in the backend as well, instead of just storing a blank string.

Reply:
it throws error, shud i revert